apiserver/facades/client/controller/controller.go

```diff
@@ -29,7 +29,6 @@ import (
 	"github.com/juju/juju/core/leadership"
 	corelogger "github.com/juju/juju/core/logger"
 	coremigration "github.com/juju/juju/core/migration"
-	"github.com/juju/juju/core/model"
 	coremodel "github.com/juju/juju/core/model"
```

Comment:
Weird that this one switched in favour of the alias? Especially since both are imported

Reply:
This one was because there were a bunch of variable name clashes with `model`, so I decided to use `coremodel` instead